spec/views/users_show_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   scenario 'Clicking on a post title redirects to showing more more details about post page' do
-     click_on("Post ##{post.id}")
-     expect(current_path).to eq(user_posts_path(user_id: post.author_id, id: post.id))
-   end
+    click_on("Post ##{post.id}")
```

Comment:
- [ OPTIONAL ] I think all the tests are failing on my side I think it would be great if you work more on them to fix them and to improve the tests as I mentioned in it blocks ⭐🔥💯 and I am assiging as optional because the previous reviewer didn't mention it and I am not sure if it just on my side but if it is in your side I highly recommend to fix them 💯⭐🔥

- Here is the message error that shown to me

![image](https://github.com/mudasiri/mybloggerapp/assets/58553711/7db5d2c8-553a-40f7-8e40-86f5dce07ffe)


![image](https://github.com/mudasiri/mybloggerapp/assets/58553711/6de91ffd-98c3-48c0-bdcb-44ccf0faa24f)